DeAceroDLabs/marketplace-frontend
# Feature/dmp 708 About us content

## What is done

[[DMP-708] About us content](https://deacero.atlassian.net/browse/DMP-708)

## How is done

- Added underlined prop to  `MailTo` component
- Updated contact mail.
- Added `text-link` color.

## How to test

1. Run the project via `npm start`.
2. Click on footer `Conócenos`.
3. You should the updated content.
4. You should see the email updated on about us view and in Contactar soporte.

## Evidence
Before:
![image](https://user-images.githubusercontent.com/81311646/203405392-b79ff20a-e329-4869-ba94-9f4ea589d51a.png)


After:
![image](https://user-images.githubusercontent.com/81311646/203405307-7d4d011e-4e56-4865-bbb7-f3abb1323973.png)
